README.md

```diff
@@ -88,12 +88,12 @@ spec to `/spec`. If you'd like to serve the spec off a path instead of a domain
 to the `hugo -d "spec"` command.
 
 For building the OpenAPI definitions, create a python3 virtualenv and activate it. Then run `pip install -r ./scripts/requirements.txt`
-and finally `python ./scripts/dump-swagger.py` to generate it to `./scripts/swagger/api-docs.json`. To make use of the generated file,
+and finally `python ./scripts/dump-openapi.py` to generate it to `./scripts/openapi/api-docs.json`. To make use of the generated file,
```

Comment:
looks like dump-openapi still writes to `./scripts/swagger`: https://github.com/matrix-org/matrix-spec/blob/217fbbeab11645bfecfc1c41a48dc00a7a7da054/scripts/dump-openapi.py#L116